_source/_includes/log-shipping/multiline-logs-filebeat.md

```diff
@@ -0,0 +1,81 @@
+## Multiline logs
+
+
+If your original logs span multiple lines, you may find that they arrive in your Logz.io account split into several partial logs. You have the option to use either an explicit configuration or an annotation to configure Filebeat to concatenate multiline logs and avoid having them separated into multiple log documents.
+
+You will need to provide a regex expression that can identify the first log in a multiline log. Filebeat will concatenate the log lines that follow until it identifies the next log that matches the regex expression. In other words, there is no explicit regex expression to demarcate the _end_ of a multiline log.
+
```

Comment:
> Filebeat will concatenate the log lines that follow until it identifies the next log that matches the regex expression.

That's not accurate, for the example this will be the behaviour, but it's because we set it to work that way, it's not the default behaviour.

```suggestion
You will need to provide a regex expression that can identify the first log in a multiline log. Filebeat will concatenate the log lines based on your configuration settings. In other words, there is no explicit regex expression to demarcate the _end_ of a multiline log.

```
or something like that?